### PR TITLE
Fix example generation for all examples

### PR DESCRIPTION
**Pull request by nlordell**
Some of the examples had hard coded values like deposits, owners, tokenIdsForPrice, etc which could also be generated to help streamline the example generation and make errors less likely.

Fixed `shortRingBetterTrade` example not actually being generated but hard coded.

Generated solutions now only include touched orders

Split up the test case generation file into smaller pieces.

Basically cleans up the new unit test generation stuff that was recently added.

### Test plan
Unit tests still pass :)

### CODE REVIEW

**Comment by bh2smith** [comment]
Amazing! Looks like only linter isn't passing. I will build on top of this soon and refactor our solidity tests! Really nice.

**Comment by nlordell** [comment]
I always forget to run the linters... Should be fixed now :)

**Review by josojo — APPROVED** [review]
This is amazing. 

Everything becomes soo clean. I am overwhelmed :)

**Comment by bh2smith** [comment]
> I always forget to run the linters

Would you like a copy of my pre-commit hook?

**Review by bh2smith — APPROVED** [review]
Amazing! Great work! Lets get it merged!

**Comment by nlordell** [comment]
@bh2smith uh-oh; there will be a failing test case cause we both merged at the same time. I have a fix, let me create a hotfix PR to address it.

**Comment by bh2smith** [comment]
Sorry! Just got too excited I guess

File: test/resources/examples/index.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    },
+    {
+      name: "Biggie Small Trade",
+      prices: [toETH(1), feeAdded(toETH(184)), ZERO],
+      buyVolumes: [ZERO, ZERO, ZERO, toETH(1), toETH(184)],
+    },
+  ],
+})
+
+const fiveThouzy = new BN("5000")
```
Comment (fleupold) [inline test/resources/examples/index.js:111]: <a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2F3NeRncMrUNb8astzVy%2Fgiphy.gif&token=joker" data-gitmeme-token="joker"></a>

Comment (bh2smith) [reply test/resources/examples/index.js:111]: Glad you like it.

Comment (fleupold) [reply test/resources/examples/index.js:111]: <a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FO5xChSjqUIxsk%2Fgiphy.gif&token=not" data-gitmeme-token="not"></a>

Comment (bh2smith) [reply test/resources/examples/index.js:111]: I suppose you'd like me to change it. The most appropriate place would be here:

#301

**Pull request merged**